packages/smooth_app/lib/helpers/analytics_helper.dart

```diff
@@ -114,7 +114,14 @@ class AnalyticsHelper {
 
   static Future<void> setAnalyticsReports(final bool allow) async {
     _allow = allow;
-    await MatomoTracker.instance.setOptOut(optout: false);
+
+    // F-Droid special case
+    if(appFlavour == 'zxing-uri' && !allow){
```

Comment:
I think you should make this a constant.
Also I'm not sure you format the code well (no space) before the bracket